src/app/ReadHandler.h

```diff
@@ -99,6 +100,19 @@ class ReadHandler
 
     virtual ~ReadHandler() = default;
 
+    CHIP_ERROR GetProcessingClusterInfo(ClusterInfo *& apClusterInfo);
+    ClusterInfo * GetClusterInfoList() { return mpClusterInfoList; };
```

Comment:
How is the lifetime of this handled? Shrink seems to remove from this item and Bump (?) increases it.